### ISSUE TITLE
Markbot keeps reading an error that I have corrected already.

### ISSUE THREAD

**Issue by DDesousa0517** [DDesousa0517/content-semantics#1]
I have corrected the indentation on the following line items but Markbot continues to tell me they are incorrect. Not sure what else I can do. @thomasjbradley

            <ol>
                <li>Albino</li>
                <li>Bull's Blood</li>
                <li>Chioggia</li>
                <li>Detroit Dark Red</li>
                <li>Early Wonder</li>
            </ol>
            <p>Serving size:<data value="100">100 g</data></p>
            <dl>
                <dt>Energy:</dt>
                <dd><data value="180">180 kJ</data></dd>
                <dt>Carbohydrates:</dt>
                <dd><data value="9.96">9.96 g</data></dd>
                <dt>Fat:</dt>
                <dd><data value="0.18">0.18 g</data></dd>
                <dt>Protein:</dt>
                <dd><data value="1.68">1.68 g</data></dd>
            </dl>
            <ul>
                <li>Crimson</li>
                <li>Awesome</li>
                <li>Pickle-able</li>
                <li>Vegetable!</li>
                <li>Delicious</li>
            </ul>

Thanks,
Darcie

**Comment by thomasjbradley** [comment]
Yeah, I see the problem. When you press the `Tab` key that puts in `2` spaces.

If you look in your code editor:
- Every dot represents 1 space
- Every vertical line represents 2 spaces

So in many of those places it looks like there are double the amount of spaces because maybe `Tab` was pressed too many times.

**Comment by DDesousa0517** [comment]
Thanks so much. I will work on correcting that now… is this normal for tab to make two spaces? Should I do something so that it only makes one?

Thanks again,
Darcie

**Comment by thomasjbradley** [comment]
It's normal—in fact it was one of the settings we configured when we were setting up Atom a few weeks ago.
